.vib/spring-cloud-skipper-shell/goss/vars.yaml

```diff
@@ -0,0 +1,7 @@
+binaries:
+  - java
+directories:
+  - mode: "0775"
+    paths:
+    - /opt/bitnami/spring-cloud-skipper-shell
```

Comment:
We are not creating or configuring the permissions for this folder, as the application 'spring-cloud-dataflow-shell' does not have a postunpack.sh logic, could you please remove these lines?